files/zh-cn/web/api/inputevent/index.md

```diff
@@ -3,29 +3,36 @@ title: InputEvent
 slug: Web/API/InputEvent
```

Comment:
Please add the meta for l10n, see the [guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE).
```suggestion
slug: Web/API/InputEvent
l10n:
  sourceCommit: 8ca15ba7933a738cf632a9bbf5cfd4e90d1a97b1
```

Reply:
thanks😀